services/credits-service/src/services/tigerbeetle_client.py

```
@@ -0,0 +1,232 @@
+                flags=AccountFlags.NONE,
+                timestamp=0,  # Let TigerBeetle set the timestamp
+            )
+
+            errors = await client.create_accounts([account])
+
+            if errors:
+                error = errors[0]
+                # Check for account already exists errors (21 = EXISTS, 25 = exists_with_different_flags)
+                if error.result in (21, 25):
+                    logger.warning(f"Account {account_id} already exists")
+                    raise AccountAlreadyExistsException(f"Account for user {user_id} already exists")
```

Ruff (0.14.5)

Abstract `raise` to an inner function

(TRY301)